<?xml version="1.0"?>
<testsuites>
 <testsuite name="CalculatorTesting.feature" package="gherkin.suite" timestamp="2016-06-15 11:19:11.011 -0600"
 tests="1" successes="1" errors="9" time="0.240000">
  <properties>
   <property name="RunDate" value="2016-06-15 11:19:11.011 -0600"/>
   <property name="Status" value="Failure"/>
  </properties>
  <testcase name="CalculatorTesting.feature" assertions="10" successes="1" errors="9" time="0.226">
    <failure type="LogError">that the Calculator app is running and in focus</failure>
    <failure type="LogError">there is no current value in the calculator</failure>
    <failure type="LogError">I enter 2 + 2 in the calculator</failure>
    <failure type="LogError">that the Calculator app is running and in focus</failure>
    <failure type="LogError">there is no current value in the calculator</failure>
    <failure type="LogError">I enter 10 minus 4 in the calculator</failure>
    <failure type="LogError">that the Calculator app is running and in focus</failure>
    <failure type="LogError">there is no current value in the calculator</failure>
  </testcase>
 </testsuite>
</testsuites>
